docs/README.tvOS.md

```diff
@@ -103,140 +109,141 @@ make -j$(getconf _NPROCESSORS_ONLN)
 ./configure --host=arm-apple-darwin --with-platform=tvos --with-sdk=11.0
 ```
 
-**[back to top](#table-of-contents)** | **[back to section top](#4-configure-and-build-tools-and-dependencies)**
+**[back to top](#table-of-contents)**
 
-## 5. Build binary add-ons
+## 5. Generate Kodi Build files
+Before you can use Xcode to build Kodi, the Xcode project has to be generated with CMake. CMake is built as part of the dependencies and doesn't have to be installed separately. A toolchain file is also generated and is used to configure CMake.
+Default behaviour will not build binary addons. To add addons to your build go to **[Add Binary Addons to Project](#52-Add-Binary-Addons-to-Project)**
 
-**NOTE:** The below is not currently possible in Kodi master repo
+## 5.1. Generate XCode Project Files
 
-You can find a complete list of available binary add-ons **[here](https://github.com/xbmc/repo-binary-addons)**.
+Before you can use Xcode to build Kodi, the Xcode project has to be generated with CMake. CMake is built as part of the dependencies and doesn't have to be installed separately. A toolchain file is also generated and is used to configure CMake.
 
-Change to Kodi's source code directory:
+Create an out-of-source build directory:
 ```
-cd $HOME/kodi
+mkdir $HOME/kodi-build
 ```
 
-Build all add-ons:
+Generate Xcode project for TVOS:
 ```
-make -C tools/depends/target/binary-addons
+make -C tools/depends/target/cmakebuildsys BUILD_DIR=$HOME/kodi-build
 ```
 
-Build specific add-ons:
-```
-make -C tools/depends/target/binary-addons ADDONS="audioencoder.flac pvr.vdr.vnsi audiodecoder.snesapu"
-```
+**TIP:** BUILD_DIR can be omitted, and project will be created in $HOME/kodi/build
```

Comment:
I just tried it out: The project has been created in `./build`.

Reply:
Not sure I see the problem?

All conventions in the readme follow on from prior steps, so if you set it up as described, Kodi source gets cloned into $HOME/Kodi, which is where you execute the cmakebuildsys command. This is then going to be ./build, as you are in the directory, or long form $HOME/Kodi/build

Reply:
You are right, I'm sorry. I've missunderstood the sentence, thought `$HOME/kodi/build` was hard coded somewere.